app/views/layouts/application.html.slim

```diff
@@ -31,15 +31,9 @@ html.govuk-template lang='en'
 
     = navigation
 
+    = render 'layouts/hero'
+    
     .govuk-width-container
```

Comment:
Out of curiosity do we need to use classes prefixed with `.dfe-..` now, rather than `.govuk-...` ?

Reply:
I'm not sure where to draw the line until there is some clarity about the design systems and 'branding' - I'm using the classes rick used for now.